### PR TITLE
Add Github actions build process

### PR DESCRIPTION
**Pull request by zhyatt**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+      - run: pip install mkdocs
+      - run: pip install mkdocs-material
+      - run: pip install pymdown-extensions
+      - run: pip install MarkdownHighlight
+      - run: pip install mkdocs-redirects
```
Comment (guilhermelawless) [inline .github/workflows/build.yml:16]: Should the python dependencies be installed from `requirements.txt`? I notice there are more in there then here, can we trim the file and use it?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+name: ci
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
```
Comment (guilhermelawless) [inline .github/workflows/build.yml:9]: We should use commit hashes such as [this example](https://github.com/nanocurrency/nano-work-server/blob/master/.github/workflows/build.yml#L28)

**Review by guilhermelawless — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (zhyatt) [reply .github/workflows/build.yml]: Done

File: .github/workflows/build.yml (hunk 2 above)
Comment (zhyatt) [reply .github/workflows/build.yml]: Done

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+name: ci
```
Comment (argakiig) [inline .github/workflows/deploy.yml:1]: Would likely change this or the other name so it's easier to differentiate in actions

**Review by argakiig — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy.yml (hunk 3 above)
Comment (zhyatt) [reply .github/workflows/deploy.yml]: Good call, done!

**Review by argakiig — APPROVED** [review]

**Review by guilhermelawless — APPROVED** [review]

**Pull request merged**